pynamodb/pagination.py

```diff
@@ -1,27 +1,75 @@
-from pynamodb.constants import CAMEL_COUNT, ITEMS, LAST_EVALUATED_KEY
+from pynamodb.constants import CAMEL_COUNT, ITEMS, LAST_EVALUATED_KEY, SCANNED_COUNT
 
 
-class ResultIterator(object):
+class PageIterator(object):
```

Comment:
you don't think people are using the class as a type hint or something in py3? we reserving this for 4.0?

Reply:
the `ResultIterator` class is still there -- the diff is just misleading -- look below